frontend/app/component/election/ElectionStatusWithIcon.test.tsx

```diff
@@ -0,0 +1,102 @@
+import { render } from "@testing-library/react";
+import { describe, expect, test } from "vitest";
+
+import { ElectionStatusWithIcon } from "./ElectionStatusWithIcon";
+
+describe("ElectionStatusWithIcon", () => {
+  test("renders the correct header coordinator status for DataEntryInProgress", () => {
+    const { getByText, getByRole } = render(ElectionStatusWithIcon("DataEntryInProgress", true, true));
```

Comment:
It's recommended to use `screen` instead of destructuring the `render`: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen.

Reply:
Updated in b514939d3a57324737f2b1e833db436db664ad6d